lib/tool_belt/commands/check_deprecation_warnings.rb

```diff
@@ -0,0 +1,68 @@
+require 'parser/current'
+require_relative '../deprecation_warning_parser.rb'
+
+module ToolBelt
+  module Command
+    class CheckDeprecationWarningsCommand < Clamp::Command
+
+      parameter "repo_path", "Path (relative to tool_belt) of repository to check"
+      option "--version", "VERSION", "Release version to check for", required: true
+      option "--paths", "PATHS", 
+             "Patterns within the repository to search, separated by a comma.",
+             default: "app/**/*.rb"
```

Comment:
Deprecation warnings could presumably be outside of app. How about checking all .rb files?

Reply:
I'd say it's ok to do only `{app,lib}/**`

Reply:
I tried that and did see some utf-8 errors when checking all of the files, I didn't look too closely but can dig in a bit more as to why that was happening. 

However, I think having a select subset will keep the unnecessary errors to a minimum. App + lib (I'll add lib to the default) should be enough to cover us as that is the where majority if not all of our business logic lives. The release owner can pass in more if they would like or have a plugin with a certain structure. If there are more folders you think we should include by default, we can add them.